pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "dictionaryutils"
+version = "3.0.3"
+description = "Python wrapper and metaschema for datadictionary."
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+
+[tool.poetry.dev-dependencies]
+nose = ">=1.3.7<2.0.0"
+rstr = "2.2.6"
+cdisutils = {git = "https://github.com/NCI-GDC/cdisutils.git", rev = "1.4.0"}
+cdislogging = "^1.0.0"
+data-simulator = {git = "https://github.com/uc-cdis/data-simulator.git", rev = "1.2.0"}
+gen3datamodel = "3.0.0"
+pytest = "^5.4.3"
```

Comment:
i don't think we need pytest?

Reply:
Yeah thats what I thought too but for some reason removing that gives me a `ModuleNotFoundError: No module named 'dictionaryutils'

Reply:
wut

Reply:
wut is right. Im not sure how that works but I get a No module for `data-simulator` as well.